backend/build_api_server.sh

```diff
@@ -76,16 +77,43 @@ if [[ ${USE_REMOTE_BUILD} == true ]]; then
   fi
 
   GCP_CREDENTIALS="$(gsutil cat ${GCP_CREDENTIALS_FILE})"
-  docker build \
-    -t "${IMAGE_TAG}" \
-    -f backend/Dockerfile \
-    . \
-    --build-arg use_remote_build=true \
-    --build-arg google_application_credentials="${GCP_CREDENTIALS}"
+  if [ $MACHINE_ARCH == "aarch64" ]; then
+    docker build \
+      -t bazel:0.24.0 \
```

Comment:
Should the tag be `bazel:0.24.0-arm64` (or `bazel:0.24.0-aarch64`)?

Reply:
Well.....Personally I think there is nothing in the Dockerfile that is specific to arm64 architecture, and it is just a building and installation of bazel. Referencing [PR#2233](https://github.com/kubeflow/pipelines/pull/2233), if this Dockerfile is compiled on ppc64le, then it will be `bazel:0.24.0-ppc64le`, and thus I didn't specific machine architecture.

However if you want its name to be `bazel:0.24.0-arm64`, I can also change. :)

Also, I think just in terms of file's name, `arm64` is the same as `aarch64`. On arm64 machines, `arm64` can be printed by shell command `dpkg --print-architecture` and `aarch64` can be printed by `uname -m`.